packages/cozy-harvest-lib/src/components/AccountForm/index.jsx

```diff
@@ -32,6 +33,14 @@ export class AccountForm extends PureComponent {
     if (locales && lang) {
       extend(locales[lang])
     }
+
+    // Reference to container element
+    this.container = null
```

Comment:
this.inputs = null 

Toujours bon de déclarer toutes ces variables de classe :)